src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2037,6 +2039,7 @@ because xbuild doesn't support framework reference assemblies.
     Condition="'$(AndroidEnableMultiDex)' == 'True' And '$(AndroidCustomMainDexListFile)' == ''"
     ToolPath="$(MainDexClassesToolPath)"
     ToolExe="$(MainDexClassesToolExe)"
+    ProguardHome="$(_MonoAndroidToolsDirectory)proguard"
```

Comment:
I think this should be a new "public" MSBuild property, e.g. `$(ProguardToolPath)`, so that alternate versions can be overridden.